### DISCUSSION TITLE
Please. Switch to the new notification badge

### DISCUSSION THREAD

**Discussion by Dhyfer1** [martinet101/ElevenClock#1240] [General]
Hi @marticliment 

In the latest version of the Windows 11 dev channel, the notifications icon is no longer circular, it's now a bell icon 🔔

This is how it appears when there are no notifications

![image_1.png](https://github.com/marticliment/ElevenClock/assets/50301201/5bb2e474-7328-4f77-97e8-0a564256dc61)

And this is how it appears when there are notifications

![image_2 (1).png](https://github.com/marticliment/ElevenClock/assets/50301201/adcac5fd-8e17-43c0-927a-1a56430e94a9)

So, I would like Elevenclock to show the new notification icon, that's all.

**Comment by marticliment** [comment]
I am aware of it. I also run Windows Insider, and I am working on the implementation of this. This feature will be available on the next update, soon (mid-september or earlier)

**Reply by Dhyfer1** [reply]
Great, I hope the new update comes as soon as possible with the new icon, and also fixed the bug where custom text appears black on light wallpapers, as I mentioned in [#1233](https://github.com/marticliment/ElevenClock/issues/1233)

**Comment by f14tomcat2** [comment]
> I am aware of it. I also run Windows Insider, and I am working on the implementation of this. This feature will be available on the next update, soon (mid-september or earlier)

The new bell icon has now made it to RTM thru the Preview Update.  I had to stop using Elevenclock.  The display got garbled.  Hope it comes out soon.  Thanks

**Comment by f14tomcat2** [comment]
@marticliment, could you give us an estimate for the next release?  I's really like to resume using it after the Icon notification problem is fixed.

Thanks.

**Reply by marticliment** [reply]
hello @f14tomcat2, I will let you know when this issue is fixed. the eta is 3 or 4 days

**Comment by marticliment** [comment]
Hello,

Please do update ElevenClock to version 4.3.1 to get this issue fixed.

Best regards,

Martí Climent

**Reply by Dhyfer1** [reply]
This new version returns the circular notification icon that was there before. I reinstalled elevenclock and the notification badge is still circular. 🫤

**Reply by marticliment** [reply]
I will need your log and your windows version

**Reply by Dhyfer1** [reply]
OK. I attach my log and I am using Windows 11 22H2 Dev channel build 23565.1000

[log.txt](https://github.com/marticliment/ElevenClock/files/13034906/log.txt)

**Reply by marticliment** [reply]
Please send me a screenshot of your "Experimental Features" section on ElevenClock settings

**Reply by Dhyfer1** [reply]
Of course. Here it is. 

![image_5](https://github.com/marticliment/ElevenClock/assets/50301201/002a2533-2f7d-4376-8618-78d41d8d6a81)

I must say that even if I disable the "Enable the zero..." setting and restart elevenclock, the circular icon still appears.

**Reply by marticliment** [reply]
I see what the issue is.

I will publish an update soon

**Reply by Dhyfer1** [reply]
Great, thank you. I look forward to the new update very soon.

**Reply by marticliment** [reply]
New ElevenClock version available

**Reply by Dhyfer1** [reply]
Thank you so much. Works as expected 😃
